packages/firebase_crashlytics/firebase_crashlytics/pubspec.yaml

```diff
@@ -10,17 +10,17 @@ environment:
   flutter: ">=1.12.13+hotfix.5 <2.0.0"
 
 dependencies:
```

Comment:
Dependencies/dev deps here need switching to use nullsafety versions, pedantic, test, firebase_core, stack_trace, firebase_core_platform_interface etc